Documentation/Signoffs/Create DAQ Subsystem.md

```diff
@@ -0,0 +1,109 @@
+# DAQ Signoff
+
+## Functionality
+
+The primary function of the data acquisition (DAQ) is capturing and displaying relevant testing data in real-time. The DAQ resolves this constraint through a process containing signal conditioners, signal converters, a controller, and software. Through the execution of correctly applying these systems, accurate, real-time data will be derived to support the generation of a model.
+
+## Constraints
+|No.        |Constraints   |Origin   |
+|---|---------------------|--------|
+|3|The DAQ must have at least 10 analog inputs, to cover all possible sensors being used from the sensor integration system.         |Design Constraint |
+|16|Analog input pins must be able to read an output of 0-5 V, 0-10 V, or 4-20mA input from the sensors.       |Derived from further considerations|
+|17|The DAQ must have an analog-to-digital converter with a minimum of 10-bit resolution.       |Derived from further considerations|
+|18|The DAQ must have a sampling rate of at least 4.8kS/s       |Derived from further considerations|
+|19|The DAQ must be able to read the RPM sensor in increments of 1 rpm, voltage in increments of 0.5 V, current in increments of 0.1 A, and torque in incremets of 0.1 N-m.      |Derived from further considerations|
+
+
+3.) To make sure all possible sensors are accounted for, it is vital to have a minimum of 10 analog inputs.
+
+16.) An industry standard for sending a signal from a sensor to a data acquisition system or controller is a 4-20 mA current loop or 0-5v voltage.  Because of this, the daq selected must be able to read in either 4-20mA, 0-5v, or 0-10v sensors on an analog input pin.
+
+17.)  Using the resolution equation in the analysis, a 10-bit ADC would allow the team to meet the minimum specified resolution. 
+
+18.) Based on different DAQ applications, it is vital that the DAQ system operates at a minimum of 4800S/s.
+
+19.)  The values must be displayed with these increments to ensure that the values that are displayed are accurate enough.  These values are determined by the calculations for each of the sensors, along with the values that are given by the daq.
+
+
+
+
+## Schematics
+
+Note: Due to the braking subsystem, the sensors are subject to change. However, if changes in the braking subsystem occur, there are plenty of analog inputs available for added sensors. Adding additional power for sensors will be addressed based on specific sensors in the sensor integration subsystem.
+
+### NI USB-6215 Wiring Diagram
+
+
+![image](https://github.com/Dylan2432/Capstone1_Team3_EV-Motorcycle-Chassis-Dynamometer/assets/157453598/2a84bf06-5fef-403f-88f1-9fcf9d6aa0c7)
+
+
+
+## Analysis
+
+### USB-6215:
```

Comment:
Could you use something less expensive to minimize cost? May be the USB-6003????